front/cypress/e2e/project_description_builder/project_description_builder_toggle.cy.ts

```diff
@@ -55,7 +55,7 @@ describe('Project description builder toggle', () => {
     projectId = '';
   });
 
-  it('shows original description by default along with any attachments if project description builder is not used', () => {
+  it.skip('shows original description by default along with any attachments if project description builder is not used', () => {
```

Comment:
Seems the attachment is the problematic thing here -could we maybe comment it put and keep the "original description" test?